miso-web/src/main/webapp/scripts/lims.js

```diff
@@ -148,6 +148,74 @@ var Utils = Utils || {
       }
     }
   },
+  
+  showOkDialog: function(title, fields) {
+    var dialogArea = document.getElementById('dialog');
+    while (dialogArea.hasChildNodes()) {
+      dialogArea.removeChild(dialogArea.lastChild);
+    }
+  
+    var output = {};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'output' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)